api/serverpb/server.proto

```diff
@@ -218,4 +256,25 @@ service Server {
       body: "*"
     };
   }
+  // SignUp creates user Percona profile.
+  rpc SignUp(SignUpRequest) returns (SignUpResponse) {
+    option (google.api.http) = {
+      post: "/v1/Auth/SignUp"
+      body: "*"
+    };
+  }
+  // SignIn creates user session.
+  rpc SignIn(SignInRequest) returns (SignInResponse) {
+    option (google.api.http) = {
+      post: "/v1/Auth/SignIn"
+      body: "*"
+    };
+  }
+  // RefreshSession refreshes session timeout.
+  rpc RefreshSession(RefreshSessionRequest) returns (RefreshSessionResponse) {
+    option (google.api.http) = {
+      post: "/v1/Auth/RefreshSession"
```

Comment:
There shouldn't be a need to expose that API – once user signed in, pmm-managed should refresh session in the background.

Reply:
Removed